packages/react-microlink/src/components/Card/CardMedia/image.js

```diff
@@ -10,20 +10,39 @@ const largeStyle = css`
   }
 `
 
+const mobileStyle = media.mobile`
+  flex: 0 0 92px;
+`
+
 export default styled.div`
   display: block;
   flex: 0 0 125px;
   background: #e1e8ed no-repeat center center / cover;
+  transition: flex-basis .25s ease-in-out;
+  position: relative;
 
-  &::before {
+  &:before, &:after {
     content: '';
+  }
+
+  &:before {
     padding-bottom: 100%;
     display: block;
   }
 
+  &:after {
+    position: absolute;
+    left: 0;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    background: #e1e8ed;
+    transition: opacity .3s ease-out;
+    ${({loading}) => loading ? `opacity: 1;` : `opacity: 0;`}
```

Comment:
You can extract `opacity` from the variable:

```
opacity: ${({loading}) => loading ? 1 : 0};
```

Reply:
Thanks, just realised that while sorting #80 😅